pandas/io/parsers.py

```diff
@@ -1569,11 +1544,6 @@ def _convert_to_ndarrays(self, dct, na_values, na_fvalues, verbose=False,
                 if cast_type and not is_dtype_equal(cvals, cast_type):
                     cvals = self._cast_types(cvals, cast_type, c)
 
-            if issubclass(cvals.dtype.type, np.integer) and self.compact_ints:
```

Comment:
I believe we can also blow away ``lib.downcast_int64``, ``to_numeric`` uses a different (and better) mechanism for doing this. IIRC ``downcast_int64`` was only for compatibility.

Reply:
GitHub search seems to verify that belief.